tests/test_sir.py

```diff
@@ -1,42 +1,46 @@
 # pylint: disable=C0114
```

Comment:
Do we need to keep these? (I am not sure, I just added them for the dummy file so that linter did not complain for just a dummy file)